lambda_functions/archive_service/tests/unit/test_get_closed_items.py

```diff
@@ -0,0 +1,55 @@
+from core_layer.connection_handler import get_db_session
+from core_layer.model.item_model import Item
+from ml_service import GetTags
+from core_layer.handler import item_handler
+import json
+import pytest
+import os
+
+from archive_service.get_closed_items import get_closed_items
+
+def test_get_closed_items(monkeypatch):
+
+    # pre-stuff
+    monkeypatch.setenv("DBNAME", "Test")
+    os.environ["STAGE"] = "dev"
+    session = get_db_session(True, None)
+    context = None
+
+    # create item
+    item = Item()
+    item.content = "Test content"
+    item.language = "de"
+    item.status = "closed"
+    item = item_handler.create_item(item, True, session)
+
+    # create events with tags
+    event1 = {
+        "pathParameters": {
+            "item_id": item.id
+        },
+        "body": json.dumps({"tags": ['C', 'B', 'D']})
+    }
+    event2 = {
+        "pathParameters": {
+            "item_id": item.id
+        },
+        "body": json.dumps({"tags": ['B', 'C']})
+    }
+    event3 = {
+        "pathParameters": {
+            "item_id": item.id
+        },
+        "body": json.dumps({"tags": ['A', 'B']})
+    }
+
+    # post tags
+    GetTags.post_tags_for_item(event1, context, True, session)
+    GetTags.post_tags_for_item(event2, context, True, session)
+    GetTags.post_tags_for_item(event3, context, True, session)
+
+    # Check if tags are sorted by number of mentions
+    response = get_closed_items(event1, context, True, session)
+    body = response['body']
+    tags = json.loads(body)[0]['tags']
+    assert tags == ['B', 'C', 'D', 'A']
```

Comment:
Could it also be BCAD if only sorted by count or am I missing something. 
If yes, please check for both possibilities

Reply:
It's only sorted by count. As D and A both have count = 1 they are in order of submission -> D was saved first. If A was entered first, the order would be BCAD.

Reply:
But BCAD would also be a correct response, right? So, to be perfectly correct, it should be something like 
`assert tags in [[B,C,D,A],[B,C,A,D]]` (I don't know if the syntax is correct). 
However this is also a really nitpicky comment and I wouldn't really mind leaving it as is ;)